docker-testing/in-container-test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This is the test script that runs in the containers themselves.
+
+WEST=/west
+
+die() {
+    [ ! -z "$@" ] && echo "error: $@" >&2
```

Comment:
I would prefer an empty `error: `. No one should `die` without a message.

I also prefer a `die` that is fully `printf` compatible: 

```shell
die()
{
        >&2 printf '%s ERROR: ' "$0"
        # We want die() to be usable exactly like printf
        # shellcheck disable=SC2059
        >&2 printf "$@"
        exit 1
}
```

Reply:
I added an error path when there's no argument. I'd rather not use printf syntax, though.